apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManagerExtendService.java

```diff
@@ -27,8 +27,13 @@
 import org.apache.skywalking.apm.agent.core.remote.GRPCChannelStatus;
 import org.apache.skywalking.apm.agent.core.sampling.SamplingService;
 
+import java.util.Arrays;
+
 @DefaultImplementor
 public class ContextManagerExtendService implements BootService, GRPCChannelListener {
+    
+    private static String[] IGNORE_SUFFIX_ARRAY = new String[0];
```

Comment:
ContextManagerExtendService's instance is controlled by the agent core. This field should not be a `static` one, like `status`.